src/codesyncd/validators/tab_validator.ts

```diff
@@ -24,20 +15,15 @@ export class TabValidator {
 
 	}
 
-	validateRepo(tabFile: ITabYML, tabData: string, filePath: string, repoId: number) {
-		const config_utils = new ConfigUtils();
-		const repo_path = config_utils.getRepoPathByRepoId(repoId);
-		const configRepo = this.configJSON.repos[repo_path];
+	validateRepo( repoPath: string) {
+		const configRepo = this.configJSON.repos[repoPath];
 		if (!configRepo) {
-			CodeSyncLogger.info(`Removing tab: Skipping invalid tab: ${tabFile}`, "", tabData);
-			removeFile(filePath, "getTabFiles");
 			return false;
 		}
 		const repoEmail = configRepo.email;
 		// Remove tab if repo is disconnected
 		if (configRepo.is_disconnected) {
-			CodeSyncLogger.error(`Removing tab: Repo ${repo_path} is disconnected`);
-			removeFile(filePath, "getTabFiles");
```

Comment:
Why aren't we removing Tab File for a dicsonnected repo anymore?

Reply:
Good catch, fixing

Reply:
Actually, we are calling this validator in `tab_event_handler` , before creating YML file. 

So if validation fails we will return and not write to file.

Reply:
Do we need to log here?
```Removing tab: Repo ${repoPath} is disconnected```
Where we are actually skipping the event only, not removing it